addons/website_slides_survey/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -1,15 +1,39 @@
 odoo.define('website_slides_survey.fullscreen', function (require) {
     "use strict";
-    
+
     var core = require('web.core');
-    var _t = core._t;
+    var QWeb = core.qweb;
     var Fullscreen = require('website_slides.fullscreen');
-    
+
     Fullscreen.include({
```

Comment:
This should be at top level nesting and not indented, like other js files.

Reply:
Done

Reply:
Done